examples/step-31/step-31.cc

```diff
@@ -952,9 +962,9 @@ namespace Step31
     // Trilinos matrices store the sparsity pattern internally, there is no
     // need to keep the sparsity pattern around after the initialization of
     // the matrix.
-    stokes_block_sizes.resize (2);
-    stokes_block_sizes[0] = n_u;
-    stokes_block_sizes[1] = n_p;
+    stokes_partitioning.resize (2);
+    stokes_partitioning[0] = complete_index_set (n_u);
+    stokes_partitioning[1] = complete_index_set (n_p);
```

Comment:
i don't think this is what you want. shouldn't these be index sets of size `n_u+n_p` of which `n_u` or `n_p` are selected?